lightningd/channel_control.c

```diff
@@ -113,10 +113,11 @@ static void try_update_blockheight(struct lightningd *ld,
 void notify_feerate_change(struct lightningd *ld)
 {
 	struct peer *peer;
+	struct peer_node_id_map_iter it;
 
-	/* FIXME: We should notify onchaind about NORMAL fee change in case
-	 * it's going to generate more txs. */
```

Comment:
Is this comment no longer necessary?

Reply:
Technically still true, but my branch which does zero-fee anchors fixes this (onchaind no longer makes its own txs).  Overzealous removal I guess.